README.md

```diff
@@ -353,18 +353,21 @@ If you want to suppress the error (which is **different** from ignoring the poss
 
 ```ts
 // This suppresses a possible error (Ignores and doesn't re-throw)
-const [ok, _, data] = try fn()
+const [ok, , data] = try fn()
```

Comment:
During destructuring assignment, it is recommended to ignore values with holes rather than `_` because JavaScript treats underscore as a regular variable. So in this case _ would be const assigned to the error value which could lead to unexpected behavior (for example, conflicting with lodash).

See _Destructuring assignment: Ignoring some returned values_ on MDN:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#ignoring_some_returned_values